blocks/library/table2/style.scss

```diff
@@ -0,0 +1,36 @@
+.editor-visual-editor__block[data-type="core/table2"] {
```

Comment:
Ok, I have moved the table styles so they are a child of `wp-block-table-2`. I can't move the other styles though because the div with `wp-block-table-2` class is not the same div that has `.editor-visual-editor__block[data-type="core/table2"]`.